mappers/modbus-dmi/modbus.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: devices.kubeedge.io/v1alpha2
+kind: Device
+metadata:
+  name: sensor-tag-instance-01
+  labels:
+    description: TISimplelinkSensorTag
+    manufacturer: TexasInstruments
+    model: CC2650
+spec:
+  deviceModelRef:
+    name: sensor-tag-model
+  protocol:
+    modbus:
+      slaveID: 1
+    common:
+      com:
+        serialPort: '1'
+        baudRate: 115200
+        dataBits: 8
+        parity: even
+        stopBits: 1
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: ''
+            operator: In
+            values:
+              - ecs-edgenode
+  propertyVisitors:
+    - propertyName: temperature
+      modbus:
+        register: CoilRegister
```

Comment:
the register of `CoilRegister` may not be ok for `temperature`

Reply:
which register is better? HoldingRegister?